package.json

```diff
@@ -201,14 +201,11 @@
                             "request": "launch",
                             "program": "^\"\\${command:ask.debugAdapterPath}\"",
                             "args": [
-                                "--accessToken",
-                                "^\"\\${command:ask.accessToken}\"",
-                                "--skillId",
-                                "^\"\\${command:ask.skillIdFromWorkspace}\"",
-                                "--handlerName",
-                                "handler",
-                                "--skillEntryFile",
-                                "^\"\\${workspaceFolder}/lambda/index.js\""
+                                "--accessToken", "^\"\\${command:ask.accessToken}\"",
+                                "--skillId", "^\"\\${command:ask.skillIdFromWorkspace}\"",
+                                "--handlerName", "handler",
+                                "--skillEntryFile", "^\"\\${workspaceFolder}/lambda/index.js\"",
+                                "--region", "NA"
```

Comment:
This is a good start. However, since we only allow an enum set of regions, I think we can make this snippet a bit more intelligent by adding [choices](https://code.visualstudio.com/docs/editor/userdefinedsnippets#_choice)

In case we don't want to go that route, I think it is better to add some kind of comments here mentioning the possible values for the region. We can use [`LINE_COMMENT` variable](https://code.visualstudio.com/docs/editor/userdefinedsnippets#_variables) to do that ? 

What do you think?

Reply:
I like that idea of choice. Here is my presentation after the change
![Screen Shot 2020-11-11 at 3 22 39 PM](https://user-images.githubusercontent.com/48572203/98876728-bc899f80-2433-11eb-935f-0c53c458c76f.png)